robot-db/src/main/java/domain/Category.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Collection;
+
+/**
+ * Category Representation in Database
```

Comment:
Which tells me NOTHING at all about what this "category" really is and what we're representing here. Useless Javadoc.